tls/client-only/mac/ca.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+echo "Creating the CSR (certificate signing request)"
+
+## This will also be used as the DNS for the end-entity.
+echo "What is your company name?:"
+read COMPANY_NAME
+
+# Lower case company name for DNS.
+CN=$(echo $COMPANY_NAME | awk '{print tolower($0)}')
+
+#RANDLETTER=$(C tr -dc 'A-Za-z0-9' </dev/urandom | head -c 4 ; echo '')
```

Comment:
We have two commented lines out that also assign the RANDLETTER variable. Do we need to include them?

Reply:
thanks!